pxtblocks/blocklyloader.ts

```diff
@@ -1183,6 +1196,13 @@ namespace pxt.blocks {
                     pxt.log("searching for: " + searchFor);
                     updateUsedBlocks = false;
                     if (!blocks) return;
+
+                    if (blocks.length == 0) {
+                        blocklySearchLabel.innerText = lf("No search results...");
+                    } else {
+                        blocklySearchLabel.innerText = lf("{0} results matching '{1}'", blocks.length, blocklySearchInputField.value.toLowerCase());
```

Comment:
- Need to handle singular form of "results" using ``{0:s}`` I think.
- what if the value is huuuuuuuuuuuuuuuuuuuuuuge, will it overflow the UI?